app/controllers/user_journey_controller.rb

```diff
@@ -17,6 +18,19 @@ def index
     end
   end
 
+  def is_dual_running
+    if params.key?(:dual_running)
+      if @not_dual_running
+        redirect_to before_you_start_path(dual_running: @not_dual_running)
+      else
+        redirect_to :before_you_start
+      end
+    else
+      Rails.logger.info(@not_dual_running)
+      redirect_to dual_running_path, flash: { error: "Must select an option" }
```

Comment:
could the `"Must select and option"` be localized and added to the errors block?